### PR TITLE
Bugfix/modal flex

### PR DESCRIPTION
**Pull request by cvializ**
@adamjgrant @andrewconner This is to make the modal directive responsive for mobile-web. I did do checking for regressions on the various consumers of the modal directive, but keep an eye out if you see anything that looks off.

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/friends/unfriendConfirmModal.tpl.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div kf-modal>
   <div kf-basic-modal-content with-cancel="true" title="Remove connection?" with-warning action="reallyUnfriend()">
     If you remove this connection with {{friend.firstName}}:
-    <ul class="kf-unfriend-confirm-ul">
+    <ul class="kf-unfriend-confirm-ul" style="white-space: nowrap;">
```
Comment (adamjgrant) [inline kifi-backend/modules/shoebox/angular/src/friends/unfriendConfirmModal.tpl.html:4]: Why inline?

File: kifi-backend/modules/shoebox/angular/src/libraries/manageLibrary.tpl.html [hunk 2]
```diff
@@ -12,27 +12,30 @@ <h1 class="dialog-header">
...
+            <h3 class="manage-lib-heading">Description</h3>
+            <div class="manage-lib-input-area">
+              <textarea class="manage-lib-input manage-lib-description-input ng-pristine ng-valid" ng-model="library.description" name="description" placeholder="Add a description..." novalidate></textarea>
+            </div>
           </div>
 
           <div class="manage-lib-input-wrapper" ng-if="me.orgs.length">
-            <div class="manage-lib-heading"></div>&nbsp;
-            <div class="manage-lib-body">
+            <h3 class="manage-lib-heading"></h3>
```
Comment (adamjgrant) [inline kifi-backend/modules/shoebox/angular/src/libraries/manageLibrary.tpl.html:37]: Nice.

File: kifi-backend/modules/shoebox/angular/src/friends/unfriendConfirmModal.tpl.html (hunk 1 above)
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/friends/unfriendConfirmModal.tpl.html:4]: I didn't want to make a `unfriendConfirmModal.styl` file with just one rule. Do you think I should just bite the bullet and add it?

Comment (adamjgrant) [inline kifi-backend/modules/shoebox/angular/src/friends/unfriendConfirmModal.tpl.html:4]: I see what you mean. Yeah, it's a little weird. Maybe we just leave it 

and never speak of this again. 

If anyone asks questions you just say you didn't see nothin of nothin and go about your business.

**Pull request merged**